### DISCUSSION TITLE
Tabs are displayed in qgis but not in qfield

### DISCUSSION THREAD

**Discussion by Kaguenze** [opengisch/QField#5782] [Q&A]
In QGIS I have set the form layout so that the attribute fields are organized in tabs. In QGIS I can see the tabs with the attribute fields. In QField, the tabs are gone and all attribute fields are displayed directly. What can I do about it?

**Comment by VxTedxV** [comment]
At first, you can search here for other issues about tabs... 😉

**Comment by Kaguenze** [comment]
> At first, you can search here for other issues about tabs... 😉

Great idea, I haven't found the solution yet, do you know something?

**Comment by Ipickedausername** [comment]
I've never encountered this issue. I think you will have to share a test project.

**Comment by nirvn** [comment]
@Kaguenze , when you design your feature form layout with tabs, you have to make sure the root elements are _only_ containers, otherwise QField will flatten everything into a tab-less list.

E.g., this will work:
![image](https://github.com/user-attachments/assets/b4ce1558-121c-4553-a173-17d94fbfb1bd)

While this will **not** work and will result in a single list of all items:
![image](https://github.com/user-attachments/assets/cb065b59-28bd-437c-ba64-1610e2bed280)
